src/openforms/registrations/contrib/objects_api/tests/test_templatetags.py

```diff
@@ -63,6 +65,74 @@ def test_render_json_summary(self):
 
         self.assertEqual(rendered, expected)
 
+    @override_settings(ESCAPE_REGISTRATION_OUTPUT=False)
+    def test_render_json_summary_doesnt_escape_html_when_disabled(self):
+        SubmissionStepFactory.create(
+            form_step__form=self.form,
+            form_step__form_definition__slug="json-summery-step",
+            form_step__form_definition__configuration={
+                "display": "form",
+                "components": [
+                    {
+                        "key": "voornaam",
+                        "type": "textfield",
+                        "registration": {
+                            "attribute": RegistrationAttribute.initiator_voornamen,
+                        },
```

Comment:
I think this configuration doesn't actually do anything, so feel free to leave it out.

There actually is a big shortcut to create submissions with particular data:

```py
submission = SubmissionFactory.from_components(
    components_list=[{"key": "voornaam", "type": "textfield"}],
    submitted_data={"voornaam": "<script>alert();</script>"},
    form_definition_kwargs={"slug": "formstep-slug"},
    completed=True,
)
```
this will create the form, form step, submission, submission step and set the provided data.